docs/UserGuide.md

```diff
@@ -334,7 +334,8 @@ Format: `clear`
 ### Undoing previous command : `undo`
 
 Restores the patient and appointment book to the state before the previous *undoable* command was executed.
-
+`undo` command will not work if you restart Nuudle.
+ 
```

Comment:
This comment is for line 346.  
For the line:
* Multiple calls to `redo` will reverse multiple undoable command execution, starting from the most recent command.
(should be `undo` instead of `redo`)